buildenv/jenkins/parentJobTemplate

```diff
@@ -0,0 +1,163 @@
+		'ppc64le_linux',
+		's390x_linux',
+		's390x_zos',
+		'x86-32_windows',
+		'x86-64_linux',
+		'x86-64_linux_xl',
+		'x86-64_mac',
+		'x86-64_mac_xl',
+		'x86-64_windows'
+	]
+}
+
+JDK_IMPL_BASE = getBaseJDKImpl(JDK_IMPL)
+
+def ADOPTOPENJDK_REPO = "https://github.com/adoptium/aqa-tests.git"
+def ADOPTOPENJDK_BRANCH = "master"
+def PERFORMANCE_REPO = "https://github.ibm.com/runtimes/rt-performance.git"
+def PERFORMANCE_BRANCH = "main"
+def PERFORMANCE_REPO_CREDENTIAL = "83181e25-eea4-4f55-8b3e-e79615733226"
+
+// https://issues.jenkins.io/browse/JENKINS-42971 is resolved, but it seems to be a partial fix for Branch Specifier parameter.
+// Repository URL still has problem with lightweight checkout
+// Set Repository URL and Branch Specifier as build parameters by default
+// If LIGHT_WEIGHT_CHECKOUT is set to true, set Repository URL to its explicit value and keep Branch Specifier as parameter
+def ADOPTOPENJDK_REPO_BUILD_PARAM = '${ADOPTOPENJDK_REPO}'
+def ADOPTOPENJDK_BRANCH_BUILD_PARAM = '${ADOPTOPENJDK_BRANCH}'
+def PERFORMANCE_REPO_BUILD_PARAM = '${PERFORMANCE_REPO}'
+def PERFORMANCE_BRANCH_BUILD_PARAM = '${PERFORMANCE_BRANCH}'
+def SCRIPT_PATH = "rt-performance/jenkins/perfPipeline.groovy"
+def LIGHT_WEIGHT_CHECKOUT = true
+
+if (LIGHT_WEIGHT_CHECKOUT) {
+	PERFORMANCE_REPO_BUILD_PARAM = PERFORMANCE_REPO
+	SCRIPT_PATH = "jenkins/perfPipeline.groovy"
+}
+
+ARCH_OS_LIST.each { ARCH_OS ->
+	def ACTUAL_TEST_JOB_NAME = TEST_JOB_NAME
+
+	pipelineJob("$ACTUAL_TEST_JOB_NAME") {
+ 		description('<h1>THIS IS AN AUTOMATICALLY GENERATED JOB. PLEASE DO NOT MODIFY, IT WILL BE OVERWRITTEN.</h1><br><p>This job is defined in <a href="https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/" target="_blank">testJobTemplatePerf</a> in the https://github.com/adoptium/aqa-tests repo. If you wish to change the job, please modify testJobTemplatePerf script. </p><p>For more information about job parameters, please refer <a href="https://github.com/adoptium/aqa-tests/wiki/How-to-Run-a-Grinder-Build-on-Jenkins" target="_blank">How to Run a Grinder Build on Jenkins</a></p>wiki:<p><a href="https://github.com/eclipse-openj9/openj9/wiki/AQA-Lightning-Talk-Series" target="_blank">AQA Lightning Talk Series</a></p><p><a href="https://github.com/eclipse-openj9/openj9/wiki/Reproducing-Test-Failures-Locally" target="_blank">Reproducing Test Failures Locally</a></p>')
+
+		properties {
+			copyArtifactPermission {
+				projectNames('*')
+			}
+		}
+
+		definition {
+			parameters {
+				stringParam('ADOPTOPENJDK_REPO', ADOPTOPENJDK_REPO, "AdoptOpenJDK git repo. Please use ssh for zos.")
+				stringParam('ADOPTOPENJDK_BRANCH', ADOPTOPENJDK_BRANCH, "AdoptOpenJDK branch")
+				stringParam('PERFORMANCE_REPO', PERFORMANCE_REPO, "PERF git repo. For PERF test only")
+				stringParam('PERFORMANCE_BRANCH', PERFORMANCE_BRANCH, "PERF branch. For PERF test only")
+
+				stringParam('PLATFORM', "${ARCH_OS}", '''Platform to be tested:
+                    <br/>arm_linux
```

Comment:
Pretty sure we've dropped some of these platforms (arm32, win32), unless this is IBM J9 specific support?